<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预警机系列配置工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="pre_entryWidget" native="true">
      <widget class="QPushButton" name="in">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>进入主界面</string>
       </property>
      </widget>
      <widget class="QLabel" name="config2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>235</y>
         <width>560</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="mid_title">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>561</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>预警机系列配置工具</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="config1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>190</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前配置文件：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="config_c">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择配置</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>配置设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>串口设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTcp">
    <property name="title">
     <string>Tcp设置</string>
    </property>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menuSerial"/>
   <addaction name="menuTcp"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
